draft-ietf-quic-transport.md

```diff
@@ -2331,7 +2335,7 @@ An endpoint only changes the address that it sends packets to in response to the
 highest-numbered non-probing packet. This ensures that an endpoint does not send
 packets to an old peer address in the case that it receives reordered packets.
 
-In response to such a packet, an endpoint MUST start sending subsequent packets
+In response to such a packet, an endpoint MUST send all subsequent packets
```

Comment:
```suggestion
In response to such a packet, an endpoint MUST send subsequent packets
```

"all" might require more commitment than we need here.  There might be another migration later.